lib/generators/stimulus_reflex/templates/app/javascript/packs/application.js.tt

```diff
@@ -0,0 +1,13 @@
+// This file is automatically compiled by Webpack, along with any other files
+// present in this directory. You're encouraged to place your actual application logic in
+// a relevant structure within app/javascript and only use these pack files to reference
+// that code so it'll be compiled.
+
+import Rails from "@rails/ujs"
+import Turbolinks from "turbolinks"
+import * as ActiveStorage from "@rails/activestorage"
+import "channels"
+
+Rails.start()
+Turbolinks.start()
+ActiveStorage.start()
```

Comment:
```suggestion
import { Turbo } from "@hotwired/turbo-rails"
import * as ActiveStorage from "@rails/activestorage"
import "channels"

ActiveStorage.start()
```

Reply:
I want to jam that "commit" button, but... I feel like if the Rails 6.1 `new` generator creates an app that uses Turbolinks - including the `data-turbolinks` stuff in the layout - then pushing Turbo into their project is probably not the best call. It could have subtle negative outcomes.

Reply:
Agreed; technically, what I'm doing right now is a fallback if they don't have an `application.js`.

However... there's no good reason for them to not have an `application.js` when this task is run. If it's not there, I should *bail*, not attempt to fix their broken app.

Definitely a rabbit hole. Thanks for pulling me out of it.